marklogic-data-hub-central/ui-custom/src/components/SocialMedia/SocialMedia.tsx

```diff
@@ -0,0 +1,103 @@
+import React, {useContext} from 'react';
+import "./SocialMedia.scss";
+import * as IconDictionary from 'react-bootstrap-icons';
+import {getValByConfig} from '../../util/util';
+import _ from "lodash";
+
+type Props = {
+  data?: any;
+  config?: any;
+};
+
+/**
+ * Component for showing one or more values for a property in a tabular view.
+ *
+ * @component
+ * @prop {object} data - Data payload.
+ * @prop {object} config  Social media configuration object.
+ * @prop {object} config.site  Social media name configuration object.
+ * @prop {object} config.site.arrayPath  Array path to find the social media name.
+ * @prop {object} config.site.path  Path to find the social media name.
+ * @prop {object} config.url  Social media url configuration object.
+ * @prop {object} config.url.arrayPath  Array path to find the social media url.
+ * @prop {object} config.url.path  Path to find the social media url.
+ * @prop {object} config.sites  Dictionary of object with the configuration (icon,color ...) of each social media.
+ * @example
+ * {
+      "component": "SocialMedia",
+      "config": {
+        "title": "Social Media",
+        "site": {
+          "arrayPath": "result[0].extracted.person.socials.social",
+          "path": "site"
+        },
+        "url": {
+          "arrayPath": "result[0].extracted.person.socials.social",
+          "path": "address"
+        },
+        "sites": {
+          "facebook": {
+            "title": "facebook",
+            "icon": "Facebook",
+            "color": "#3B5998"
```

Comment:
In one of the examples here, can we change the size, e.g. "size": 24

Reply:
@wooldridge I added the size.